test/e2e/scripts/auth_utils.sh

```
@@ -301,13 +319,42 @@ run_auth_debug_report() {
   _append ""
 
   _section "MaaS API Deployment"
-  _run "maas-api pods" "kubectl get pods -n $DEPLOYMENT_NAMESPACE -l app.kubernetes.io/name=maas-api -o wide 2>/dev/null || true"
-  _run "maas-api service" "kubectl get svc maas-api -n $DEPLOYMENT_NAMESPACE -o wide 2>/dev/null || true"
+  _run "maas-api pods" "kubectl get pods -n $INFRA_NAMESPACE -l app.kubernetes.io/name=maas-api -o wide 2>/dev/null || true"
+  _run "maas-api service" "kubectl get svc maas-api -n $INFRA_NAMESPACE -o wide 2>/dev/null || true"
+  _run "maas-api endpoints" "kubectl get endpoints maas-api -n $INFRA_NAMESPACE -o wide 2>/dev/null || true"
+  
+  # Pod readiness details (if pods exist)
+  local pod_count
+  pod_count=$(kubectl get pods -n $INFRA_NAMESPACE -l app.kubernetes.io/name=maas-api --no-headers 2>/dev/null | wc -l || echo "0")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)